nixio/block.py

```diff
@@ -228,9 +227,9 @@ def create_data_array(self, name="", array_type="", dtype=None, shape=None,
         if copy_from:
             if not isinstance(copy_from, DataArray):
                 raise TypeError("Object to be copied is not a DataArray")
-            id = self._copy_objects(copy_from, "data_arrays",
+            objid = self._copy_objects(copy_from, "data_arrays",
                                     keep_copy_id, name)
```

Comment:
actually I am not sure, if I really think that objid is a better variable name than id ;)
``jan.set_nitpick = true``
Anyway, indentation is broken here
``jamn.set_nitpick = false``

Reply:
`id` is a builtin function in Python so I don't like using it for variable or function names.